contributors/design-proposals/propagation.md

```diff
@@ -150,6 +165,42 @@ non-privileged', an environment check and an WARNING log will be emitted about
 whether propagation mode is supported.
 
 
+## Extra Concerns
+
+@lucab and @euank has some extra concerns about pod isolation when propagation
+modes are changed, listed below:
+
+1. how to clean such pod resources (as mounts are now crossing pod boundaries,
```

Comment:
@euank, as we don't see much concern on "slave" mode, we actually have option which is not on-by-default:
> add a new option in "SecurityContext" to explicitly allow mounts in HostPath volume, only use shared mount when it is true, use slave mount otherwise.